monitor/grafana/zeebe-overview.json

```diff
@@ -537,6 +488,122 @@
         "alignLevel": 1
       }
     },
+    {
+      "datasource": null,
+      "description": "Shows the reason for the last pod restart",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "custom": {
+            "align": null,
+            "filterable": false
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
```

Comment:
:question: Do we need this?

Reply:
These are the defaults. I don't think I added it.